setup.py

```diff
@@ -1,6 +1,7 @@
 import setuptools
 from setuptools.command.build_ext import build_ext
```

Comment:
Could you remove this `import`, since it's no longer needed?

Reply:
```suggestion
```